README.md

```diff
@@ -0,0 +1,35 @@
+# Pathfinder Monster Creator
+Project supporting pen & paper RPG game design using machine learning. It offers such functionalities as calculating 
+monster's Challenge Rating (CR) based on its properties and generating new, similar monsters with a different CR value 
+modified by user.
+
+This project was created as the core of the Engineering Thesis titled "Application of machine learning to support 
+pen & paper RPG game design".
+
+## Authors
+* Paulina Jędrychowska
+* Bogumiła Papiernik
+* Jolanta Śliwa
+
+## Table of contents
+* [Technologies](#technologies)
+* [Features](#features)
+* [Project Structure](#project-structure)
+
+## Technologies
+* Python 3.8 or higher
+* Jupyter Notebook
+
+## Features
+* calculating Challenge Rating based on given properties
+* for given monster and selected different CR generating new sets of monster's properties similar to the original one
+* searching for similar monsters in Pathfinder's resources
+* suggesting hints of properties that can be added to user's monster based on similar monsters
+
+## Project Structure
```

Comment:
brak folderu na kod pythonowy (chyba, że założenie, iż wszystko będzie w tym training)

Tu przeklejam fragment rozmowy z morgulem
"config - trzyma pliki YAML (albo dotenv, jak kto woli) ze zmiennymi środowiskowymi, np. dev.yml (development lokalny), stg.yml (staging), prod.yml
docker - pliki Dockerfile, docker-compose.yml itd.
notebooks - Jupyter Notebooki
src - trzyma kod Pythonowy. W głównym repo jest tylko plik main.py z tych plików Pythonowych
test - testy"

Reply:
W bodajże kolejnej wiadomości z tego czatu: "Możecie natomiast zrobić tak, że macie np.
zamiast src katalogi training i serving, gdzie:
○ training to są skrypty, które tworzą dataset, trenują model i zwracają na dysku
zapisane pliki z wytrenowanymi modelami
○ serving to API z faktycznie działającą apką, które dostaje pliki z modelami na
dysku" - tak zrobiłam